cfme/tests/containers/test_reload_button_provider.py

```diff
@@ -42,22 +44,17 @@ def test_reload_button_provider(provider):
     list_img_from_registry_splitted_new = set(list_img_from_registry_splitted)
     list_img_from_openshift_parsed_new = set(list_img_from_openshift_parsed)
 
-    list_img_from_openshift_parsed_new.update(list_img_from_registry_splitted_new)
+    list_img_from_openshift_parsed_new.update(
+        list_img_from_registry_splitted_new)
 
     num_img_in_cfme = provider.num_image()
     # TODO Fix num_image_ui()
 
     num_img_cfme_56 = len(provider.mgmt.list_image())
     num_img_cfme_57 = len(list_img_from_openshift_parsed_new)
+    num_img_cfme_58 = len(list_img_from_openshift_parsed_new)
 
     assert num_img_in_cfme == version.pick({version.LOWEST: num_img_cfme_56,
-                                            '5.7': num_img_cfme_57})
-
-    # validate the number of image registries
```

Comment:
i presume the comment about images being tested separately refers to this removal bit?